### PR TITLE
Add k3s airgap images to build

### PR DESCRIPTION
**Pull request by dramich**
https://github.com/rancher/rancher/issues/28916

### CODE REVIEW

File: package/Dockerfile [hunk 1]
```diff
@@ -100,6 +103,8 @@ RUN curl ${HELM_URL_V3} | tar xvzf - --strip-components=1 -C /usr/bin && \
 RUN curl -sLf ${!TINI_URL} > /usr/bin/tini && \
     curl -sLf ${!CHANNELSERVER_URL} > /usr/bin/channelserver && \
     curl -sLf ${!K3S_URL} > /usr/bin/k3s && \
+    mkdir -p /var/lib/rancher/k3s/agent/images/ && \
+    curl -sLf ${!K3S_IMAGE_TAR} > /var/lib/rancher/k3s/agent/images/${CATTLE_K3S_VERSION}-images.tar && \
```
Comment (cbron) [inline package/Dockerfile:107]: Shouldn't the final path be `.../k3s-airgap-images-$ARCH.tar` ? Thats how docs have it, but idk if k3s just picks up whatever tar is in that dir.

**Review by cbron — COMMENTED** [review]
Thought we needed Norman PR to update k3s server setting, but guess not, should pick these up automatically.

File: package/Dockerfile (hunk 1 above)
Comment (dramich) [reply package/Dockerfile:107]: The name is not important, any tar file in the dir will get extracted.

**Review by cbron — APPROVED** [review]
lgtm

**Comment by ibuildthecloud** [comment]
We only really need coredns and pause.  So it's bummer to pull in a couple hundred mb for that.

**Review by ibuildthecloud — APPROVED** [review]

**Review by cbron — APPROVED** [review]
makes sense to me

**Pull request merged**